package.json

```diff
@@ -126,6 +126,7 @@
   },
   "resolutions": {
     "braces": ">=2.3.1",
-    "https-proxy-agent": ">=3.0.0"
+    "https-proxy-agent": ">=3.0.0",
+    "handlebars": ">=4.5.2"
```

Comment:
handlebars?

Reply:
https://handlebarsjs.com — `{{…}}` Templating syntax.

There was security vulnerability published 4 days ago: https://www.npmjs.com/advisories/1316

In any case whichever dependency (jest I think?) was using it has probably updated, so we don't need this resolution anymore.